govc/flags/debug.go

```diff
@@ -69,7 +69,8 @@ func (flag *DebugFlag) Process(ctx context.Context) error {
 		// Base path for storing debug logs.
 		r := os.Getenv("GOVC_DEBUG_PATH")
 		if r == "" {
-			r = home
```

Comment:
Don't want to do this as it makes `govc ... -debug` output to stderr by default.  We could do something like:
```go
switch r {
      case "-":
           debug.SetProvider(&debug.LogProvider{})
           return nil
      case "":
             r = home
}
```
Various other commands use `-` to specify stdout (stderr in this case) instead of a file.

Reply:
ya i thought this one might be dinged. ill switch to -